### ISSUE TITLE
[QUESTION] Does Nintendo can really not patch this ?

### ISSUE THREAD

**Issue by mariogamer2** [delebile/arm9loaderhax#34]
So.... When analysing the way that arm9loaderhax work on the 32C3,I get doubt on this question....

Here are the two question I want to get answer:
1. Does Nintendo can update the bootrom through their software updater? If yes,can they patch a9lh? (see question 2)
2. Is this a possibility that Nintendo add a check in the bootrom who verify the size of the NATIVE_FIRM (or his total hash,not only the header) that will "kill" a9lh?

Thank.

**Comment by ihaveamac** [comment]
bootrom can only be updated with a hardware revision. if they could update bootrom with software, that could just be blocked (and also we'd have its contents).

**Comment by pixel-stuck** [comment]
Googling the term "ROM" and figuring out what it means/stands for will
answer your question

Also, this is the issues section. Your question is NOT an issue, take this sort of thing elsewhere

**Comment by mariogamer2** [comment]
Ok thank. (I know what is a ROM,I was thinking there's still possibility since on PC we can overwrite hem (yeah it is a FlashROM...)). Sorry... (we cannot make question on github? well I already found some question,nevermind...) Thank.

**Comment by mariogamer2** [comment]
@dark-samus ,not necessarily,it depend on the type or the question. Also,there is question tag on github,too. (I already asked some question by myself,too)

Also,I think I can say this about the subject:
1. Like @dark-samus say, we cannot write a ROM through software,it must be hardware (and I know what is a ROM,I was just thinking it was some-kinda of Flash ROM (like on PC), but I was stupid at that point :P)
2. -They cannot check the hash of all the FIRM,because the hash change with a lot of software update
   -Checking the size is still a good idea but I think alt_stage2 can be used in that case.
   -They can free the memory before checking second firm...

Does it's bad?
